### ISSUE TITLE
KDD 2025 Tutorial Subpages

### ISSUE THREAD

**Issue by rvandewater** [Medical-Event-Data-Standard/medical-event-data-standard.github.io#60]
We need a KDD tutorial-ready website. We also need to consider what needs to be a Google Colab notebook and what needs to be a webpage, or how this should overlap.

 Ideally, the page should be easily accessible and include the following:

**(1) Setting the Stage: Driving Problem & Tutorial Set-up**
• Introducing the reproducibility challenge in health AI.
• Discussing how MEDS [ 6] as a data standard open-source
ecosystem can help solve this reproducibility challenge.
• Description of the guiding problem: Building a model for
a meaningful task on an EHR dataset.
• Interactive: Setup check: participants verify Jupyter note-
book access and dependencies.
**(2) What is MEDS?**
• Deep dive into MEDS data schema, core principles, inter-
operability advantages.
• Relation to other formats and advantages for ML use-cases.
• Interactive: Q&A session.
**(3) Getting Data into MEDS**
• Introduction to the MIMIC-IV Demo dataset [3].
• Explore the MEDS ETL template to see how these data
were created.
• Provide references to other MEDS ETLs on public datasets,
including eICU [11], AUMCdb [1], and NWICU [7].
• Interactive: Q&A session.
**(4) Extracting Predictive Tasks with ACES**
• Introduction to ACES configuration language [15].
– Interactive: Q&A on ACES language.
• Defining our prediction task.
– Interactive: Task extraction via ACES CLI.
**(5) Building a Model**
• Pre-processing data (normalization, tokenization, tensoriza-
tion).
– Interactive: Run pre-processing CLI tool via meds-torch-
data.
– Other preprocessing tools overview (MEDS-Transforms,
meds_reader [14], MEDS Inspect).
• Interactive: Instantiate a PyTorch dataset.
• Interactive: Build a PyTorch model.
**(6) Training & Evaluating the Model**
• Interactive: Write training code, train model.
• Interactive: Write prediction code.
• Comments on standardized evaluation.
• Interactive: Evaluate using MEDS-Evaluation.
**(7) Open Research with MEDS-DEV**
• Overview of decentralized benchmarking (MEDS-DEV).
• Interactive: Generate baseline metrics using models (MEDS-
Tab [ 9], CEHR-BERT [ 10 ], GenHPF [ 2], MOTOR [ 13 ], CORE-
BEHRT [8], ETHOS [12]).
• Interactive: Small group discussions on task definition,
benchmarking validity, contributions to MEDS-DEV
